### PR TITLE
Fix id counter for notifications

### PR DESCRIPTION
**Pull request by OHerasym**
Related:
        [APPLINK-16149](https://adc.luxoft.com/jira/browse/APPLINK-16149)

[old pull request](https://github.com/CustomSDL/sdl_atf/pull/90)

### CODE REVIEW

File: modules/mobile_session.lua [hunk 1]
```diff
@@ -13,6 +13,9 @@ local FAILED = expectations.FAILED
 local module = {}
 local mt = { __index = { } }
 mt.__index.cor_id_func_map = { }
+
+module.notification_counter = 0
```
Comment (LuxoftAKutsan) [inline modules/mobile_session.lua]: Looks like all session would have only one notification counter, isn't it?

**Comment by LuxoftAKutsan** [comment]
Please describe problem that you fixed in commit message

File: modules/mobile_session.lua (hunk 1 above)
Comment (OHerasym) [inline modules/mobile_session.lua]: @LuxoftAKutsan , Yes, correct

**Comment by OHerasym** [comment]
@LuxoftAKutsan , Done

**Comment by LuxoftAKutsan** [comment]
Reviewed. @AGaliuzov  please merge

**Pull request merged**